crud/replace.lua

```diff
@@ -90,9 +89,47 @@ function replace.call(space_name, obj, opts)
 
     local tuple = results[replicaset.uuid]
     return {
-        metadata = table.copy(space_format),
+        metadata = table.copy(space:format()),
         rows = {tuple},
     }
 end
 
+--- Insert or replace an object in the specified space
+--
+-- @function object
+--
+-- @param string space_name
+--  A space name
+--
+-- @param table obj
+--  Tuple object (according to space format)
```

Comment:
`Object`?

Reply:
Fixed in https://github.com/tarantool/crud/compare/57d8d662f964250f6720c850a734721e730144ba..0583a496d5d16035d34ddad4447cd1ca862b8788